code/string_algorithms/src/pangram_checker/README.md

```diff
@@ -0,0 +1,16 @@
+## PANGRAM
+A pangram is a sentence in which every letter of a given alphabet appears at least once. The most famous english pangram is "The quick brown fox jumps over a lazy dog.".
```

Comment:
No need for two periods in a sentence. Remove the second to last one from the end.